File: xNet/src/main/java/org/sqlunet/Has2Pos.java
/*
 * Copyright (c) 2023. Bernard Bou
 */

package org.sqlunet;

import androidx.annotation.Nullable;

/**
 * Has part-of-speech interface
 *
 * @author <a href="mailto:1313ou@gmail.com">Bernard Bou</a>
 */
public interface Has2Pos extends HasPos
{
	/**
	 * Get pos
	 *
	 * @return pos
	 */
	@Nullable
	Character getPos2();
}

File: xNet/src/main/java/org/sqlunet/Has2Pos.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.sqlunet

/**
 * Has part-of-speech interface
 *
 * @author [Bernard Bou](mailto:1313ou@gmail.com)
 */
interface Has2Pos : HasPos {

    /**
     * Get pos
     *
     * @return pos
     */
    fun getPos2(): Char?
}
